art_internal_impl.hpp

```diff
@@ -662,6 +748,195 @@ class basic_inode_impl : public ArtPolicy::header_type {
     // LCOV_EXCL_STOP
   }
 
+  // Return the child node at the specified child index.
+  //
+  // Note: For N48, the child_index is the index into the
+  // child_indices[].  For all other node types, it is a direct index
+  // into the children[].  This method hides this distinction.
+  [[nodiscard]] constexpr node_ptr get_child(node_type type, std::uint8_t child_index) noexcept {
+    UNODB_DETAIL_ASSERT(type != node_type::LEAF);
+    // Because of the parallel updates, the callees below may work on
+    // inconsistent nodes and must not assert, just produce results, which are
+    // OK to be incorrect/inconsistent as the node state will be checked before
+    // acting on them.
```

Comment:
This comment is now repeated multiple times. Let's group all the read accessors into a single section in a class, and make it a comment for that section

Reply:
Done